### DISCUSSION TITLE
Capture Sites

### DISCUSSION THREAD

**Discussion by TylorTurner** [TownyAdvanced/Towny#6854] [Q&A] [answered]
Unsure if I am doing something wrong but Towny Capture Sites seem to have friendly fire forced on? Which takes over normal Towny friendly fire. I have friendly fire toggled off in the world.

**Comment by LlmDl** [comment]
I looked at the code, are the players in the same town or the same nation?

**Comment by TylorTurner** [comment]
The players are in the same Town, I have not tested it with same Nation.
Nations are not supposed to be teams with capture sites though? I know you
have a option of attacking your own nation on capture sites. Mine allows it
as default I believe.

**Reply by LlmDl** [reply]
Weird, I am pretty sure we're only making FF a thing when its not a same-town situation. TCS listens to the FriendlyFire events because we need to make it so nations towns can fight over a capture site without combat being blocked.

I will look into it when I am back from vacation.

**Reply by LlmDl** [reply]
I've looked into this and I do not see how players in the same town would have their FriendlyFire enabled.

**Reply by TylorTurner** [reply]
Double checked it, I start a Capture Site with my team and can still hit them with friendly fire disabled. It enables it during the fight. After it's over we are back to normal and can't hit each other.

**Reply by LlmDl** [reply]
And that team is people in your own town?

What does `/tw` show you?

**Reply by TylorTurner** [reply]
Yea they are in my Town.
![2023-07-21_14 30 47](https://github.com/TownyAdvanced/Towny/assets/133236597/74812dde-689c-4841-9243-bb0adf01625d)

**Reply by LlmDl** [reply]
Alrighty, seems there's a bug somewhere.

**Reply by LlmDl** [reply]
Please try re-testing using 0.99.5.5 https://github.com/TownyAdvanced/Towny/releases/tag/0.99.5.5

**Reply by TylorTurner** [reply]
Okay will do. Also another issue possibly. When A Town that once owned a Capture Site is Deleted. That capture site is unobtainable. User receives errors. I have to TA remove defenders to fix it.
![image](https://github.com/TownyAdvanced/Towny/assets/133236597/967e05b2-db8c-4576-82de-985b21ae27ef)

**Reply by TylorTurner** [reply]
I really hope to see a separate spawn point outside the Capture Site, Also some kind of particle to tell the users where the capture site is. 
Keep inventory would be HUGE for my users as well. Possibly a spawn mechanic that forces both teams in place before the "match" starts.
Sorry for the requests, I think this idea is by far the best I have seen for a survival world.

**Reply by LlmDl** [reply, marked as answer]
https://github.com/TownyAdvanced/Towny/discussions/6854#discussioncomment-6529790 

This should be fixed in 0.15.3

**Reply by TylorTurner** [reply]
okay.
